README.md

```diff
@@ -132,9 +132,20 @@ clients = cs.clients
 clients.each do |cl|
   p "Client: #{cl.Name}"
   client = CreateSend::Client.new auth, cl.ClientID
+  campaigns = []
+  page_number = 1
+  loop do
+      page = @client.campaigns(page_number, 1000, 'desc', '', '', '')
```

Comment:
Do we need to pass through page size etc as there are default values on the function already for them?